frontend/config/webpack.config.prod.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const path = require("path");
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - Parsing error: The keyword 'const' is reserved